src/materials/Material.js

```diff
@@ -70,6 +70,7 @@ function Material() {
 	this.premultipliedAlpha = false;
 
 	this.visible = true;
+	this.parallelCompile = false;
```

Comment:
Is this boolean needed?
What are the downsides of using KHR_parallel_shader_compile by default?

Reply:
A material can be ready a few animation frames away since the material is added to the scene. That means The mesh can't show up in the first few animation frames. So if it's true as default

1. It breaks the compatibility.
2. Problematic for applications which doesn't use animation frame, for example calling renderer.render() only when something in the application is changed.

Reply:
Does it need to be per material?

Reply:
Personally I think moving the flag to renderer would be simpler.

Reply:
> What are the downsides of using KHR_parallel_shader_compile by default?

In addition to what I mentioned above, if it hasn't been updated, there is a chance that the extension may have performance issue on Chrome due to browser side problem now. (Sorry I've been delaying to evaluate and report! I'm happy if anyone evaluates.)

Reply:
I added the flag to the material mainly out of caution and the issue where either multiple objects are added to a scene simultaneously or a single object with multiple materials, which may cause unwanted glitches as the object/object parts become visible in different frames as the compilations complete.

If you don't think this would cause an issue in practice I'll make it a renderer flag.

Reply:
Yeah, may be easier to start with the renderer first, and if people need more granularity we can consider moving to deeper objects.

Reply:
What is your preferred API for a renderer swicth?

A new property of the parameters object or a simple boolean parameter of the renderer itself?

Reply:
OK,  I have found out an issue with a renderer wide switch.

Some passes like the PMREMGenerator carry out single pass operations which can and do fail with parallel compilation enabled when the compilation does not complete immediately, I imagine there are other cases out there.

Reply:
I see...

Reply:
I could set a global default on Material.DefaultParallelCompile and use that to init each material.

So a user could opt in globally for simplicity but the default would be safe for all the odd corner cases out there.